src/Input/Mouse.cpp

```diff
@@ -125,39 +136,35 @@ void Mouse::_setType(Cursor state)
             break;
         case Cursor::SCROLL_W:
             _ui = new UI::Image("art/intrface/scrwest.frm");
-            _ui->setYOffset( -_ui->height()*0.5);
+            _ui->setOffset(0, -_ui->size().height() / 2);
```

Comment:
Please be more specific (line #?) I looked through changes in the switch/case and didn't find any difference in logic.